sdk/core/azure-core/test/ut/extendable_enumeration_test.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <azure/core/internal/extendable_enumeration.hpp>
+#include <gtest/gtest.h>
+#include <string>
+
+using namespace Azure::Core;
+
+class MyEnum : public Azure::Core::_internal::ExtendableEnumeration<MyEnum> {
+public:
+  MyEnum(std::string initialValue) : ExtendableEnumeration(std::move(initialValue)) {}
```

Comment:
Ditto `explicit` (less of a concern for test code, but consistency never hurts).